src/pages/layer-2.js

```diff
@@ -0,0 +1,614 @@
+  const heroContent = {
+    title: "Layer 2",
+    header: "Layer 2",
+    subtitle: "Smart solutions to scale Ethereum",
+    image: getImage(data.ethBlocks),
+    alt: "test",
+    buttons: [
+      {
+        content: "Use Layer 2",
+        pathId: "use-layer-2",
+      },
+      {
+        content: "Wait, what's Ethereum?",
+        pathId: "/",
+        isSecondary: "isSecondary",
+      },
+    ],
+  }
+
+  const layer2Cards = [
+    {
+      image: privacyGlyph,
+      title: "Inclusivity",
+      preview: "Lower fees and better user experience",
+      body: [
+        "By combining multiple transactions into a single transaction on layer 1, transaction fees are massively reduced, making Ethereum more accessible for all.",
+      ],
+      alt: "test",
+    },
+    {
+      image: privacyGlyph,
+      title: "Expand Use Cases",
+      preview: "Increase the scope of projects",
+      body: [
+        "With higher transactions per second, cheaper fees, and new technology, the scope of projects is able to expand into new use cases we have yet to see come online. Opportunities for gaming, exchanges, and applications yet to be created will be able to come online with better user experience.",
+      ],
+      alt: "test",
+    },
+    {
+      image: privacyGlyph,
+      title: "Security",
+      preview: "Layer 2 is cashing a cheque, layer 1 is going to court",
+      body: [
+        "We need need scaling solutions that inherit Ethereums security. Layer 2 networks settle their transactions on the Ethereum mainnet, allowing users who use them to benefit from the security of the Ethereum network.",
+        "Layer 2's take advantage of the consensus mechanism of its parent Layer 1, in this case Ethereum, instead of providing their own consensus mechanism.",
+      ],
+      alt: "test",
+    },
+  ]
+
+  const layer2DataCombined = [...layer2Data.optimistic, ...layer2Data.zk]
+
+  return (
+    <GappedPage>
+      <PageMetadata
+        title={"Layer 2"}
+        description={"Introduction page to layer 2"}
+      />
+
+      <HeroContainer>
+        <Hero content={heroContent} isReverse />
+      </HeroContainer>
+
+      <Content>
+        <h2>What is Layer 1?</h2>
+        <p>
+          Layer 1 blockchains, like Ethereum and Bitcoin, are the underlying
+          foundation that layer 2 projects build on top of. Some examples of
+          layer 2 projects being build on top of these networks are zk-rollups
+          and optimistic-rollups on Ethereum, and the Lighting Network on top of
+          Bitcoin.
+        </p>
+        <p>
+          Ethereum as the layer 1 includes the blockchain itself and the history
+          of transaction data, a network of miners (soon to be validators after{" "}
+          <Link to="/upgrades/merge/">The Merge</Link>) proposing blocks, a
+          network of node operators to secure and validate the network, and the
+          consensus mechanism for the network. Ethereum also function as a data
```

Comment:
Even saying "Ethereum also functions as a data availability layer" might be a bit too much jargon here. You already say it includes the history of transaction data. 

I think we keep it simple: L1 is the base layer blockchain. It holds all transaction history & therefore the entire state of the blockchain.

Reply:
Sounds good. Ill make sure to bring up data availability later in the layer 2 section, but I agree it can be later. I think introducing that term on this page should happen still even if its slightly in the weeds as it is a common phrase used when talking about layer 2 in media.